config/routes.rb

```diff
@@ -128,8 +128,10 @@
         get '/interview-preferences', to: redirect('/candidate/application/interview-needs')
         get '/interview-preferences/review', to: redirect('/candidate/application/interview-needs/review')
 
-        get '/' => 'personal_statement#edit', as: :edit_becoming_a_teacher
-        patch '/' => 'personal_statement#update'
+        get '/' => 'personal_statement#new', as: :new_becoming_a_teacher
+        patch '/' => 'personal_statement#create'
```

Comment:
should this be a `post`?

Reply:
It's actually just updating an attr on the application form though right?

Reply:
that's true, a `patch` makes sense

Reply:
Okay great. Thanks for the review @stevehook !